### PR TITLE
main/perl-test-inter: upgrade to 1.09

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

**Comment by maxice8** [comment]
@ncopa

File: main/perl-test-inter/APKBUILD [hunk 1]
```diff
@@ -2,16 +2,14 @@
...
 url="http://search.cpan.org/dist/Test-Inter/"
 arch="noarch"
-license="GPL PerlArtistic"
-cpandepends=""
-cpanmakedepends="   "
-depends="$cpandepends"
-makedepends="perl-dev $cpanmakedepends perl-module-build"
+license="GPL-1.0-or-later OR Artistic-1.0-Perl"
+makedepends="perl-dev perl-module-build"
+checkdepends="perl-test-pod perl-test-pod-coverage perl-file-find-rule"
```
Comment (tcely) [inline main/perl-test-inter/APKBUILD]: `perl-file-find-rule` is in `community` repository.

Comment (maxice8) [reply main/perl-test-inter/APKBUILD]: moved it and its dependencies to community

**Comment by tcely** [comment]
I don't see why `perl-date-manip` is involved. Would you explain that?

**Comment by maxice8** [comment]
On Sun, Jun 02, 2019 at 10:39:57PM -0700, tcely wrote:
> I don't see why `perl-date-manip` is involved. Would you explain that?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8239#issuecomment-498119490

perl-date-manip depends on perl-test-inter

**Pull request closed without merging**